diff --git a/configure.ac b/configure.ac
index a69daaf..d0a3455 100644
--- a/configure.ac
+++ b/configure.ac
@@ -61,7 +61,7 @@ AS_IF([test "x$$1" = "x"], [$3],
 # handle it.
 AC_MSG_CHECKING([whether to handle compressed pixmaps])
 case $host_os in
-        *mingw*)        zpipe_default="no" ;;
+        *mingw*|*msys*)        zpipe_default="no" ;;
         *)              zpipe_default="yes" ;;
 esac
 AC_ARG_ENABLE(open-zfile,
